### PR TITLE
[16.0][IMP] edi_oca: add partner form page

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Fix the TODO at: https://github.com/OCA/edi-framework/blob/16.0/edi_sale_oca/views/res_partner.xml#L13

### CODE REVIEW

File: edi_oca/views/res_partner.xml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2022 Camptocamp SA
```
Comment (etobella) [inline edi_oca/views/res_partner.xml]: Don't remove the copyright

**Review by etobella — CHANGES_REQUESTED** [review]
I do not agree with this change on an already installed version. This could affect other modules accross other repos (inside OCA or not). At the end, we need to prevent this kind of changes on Mature modules (I know, it says Beta, but after 4 versions, we can talk of a Mature module....). I would recommend to apply this change on  thenew release for 17 #47

**Comment by simahawk** [comment]
I think we can find a solution here: do not touch the other modules. They are adding a group in a specific section, not a page. Just move the view from edi_sale to core.
I would also add a TODO in those modules to move their conf to the same page in their own group.

**Comment by simahawk** [comment]
As you are there, please rewrite the commit as 
```
[IMP] edi_oca: add partner form page
```

**Comment by QuocDuong1306** [comment]
Thank @simahawk @etobella , I updated

File: edi_account_oca/views/res_partner.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="account.view_partner_property_form" />
         <field name="arch" type="xml">
+            <!-- TODO: Move this conf to the EDI page in edi_oca. -->
```
Comment (simahawk) [inline edi_account_oca/views/res_partner.xml]:
```suggestion
            <!-- TODO: Move this conf inside `edi` page from edi_oca. -->
```

File: edi_stock_oca/views/res_partner.xml [hunk 3]
```diff
@@ -7,6 +7,7 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="stock.view_partner_stock_form" />
         <field name="arch" type="xml">
+            <!-- TODO: Move this conf to the EDI page in edi_oca. -->
```
Comment (simahawk) [inline edi_stock_oca/views/res_partner.xml]:
```suggestion
            <!-- TODO: Move this conf inside `edi` page from edi_oca. -->
```

**Review by simahawk — APPROVED** [review]
Minor remarks, otherwise ok for me.
@etobella better?

**Review by etobella — APPROVED** [review]

**Review by etobella — APPROVED** [review]
/ocabot merge minor

**Pull request merged**